demos/tabs.scss

```diff
@@ -24,27 +24,42 @@
   @include mdc-tab-label-ink-color($material-color-indigo-500);
 }
 
-#basic-tab-bar-custom-label-color .mdc-tab:not(.mdc-tab--active),
-#css-tab-bar-custom-label-color .mdc-tab:not(.mdc-tab--active) {
-  @include mdc-tab-label-ink-color($material-color-deep-purple-200);
+#basic-tab-bar-custom-label-color,
```

Comment:
hmm, i didn't catch this in my first review, but I don't think we should be using ID CSS selectors in our demo pages. Its not what we would recommend our users to use. We would recommend they use CSS classes. So we should do the same.

so like:

````
.custom-label-color-tab {
  &:not(.mdc-tab--active) {
    @include mdc-tab-label-ink-color(purple);
  }
  &:hover {
    @include mdc-tab-label-ink-color(indigo);
  }
  &.mdc-tab--active {
    @include mdc-tab-label-ink-color(indigo);
  }
}

...and the same for all the CSS below